README.md

```diff
@@ -229,6 +229,23 @@ To increase or decrease the logging level for the S3 plugin, add an `s3_resource
 },
 ```
 
+## Troubleshooting
+
+### Handling Reduce Your Request Rate Errors
+
+On operations that require a checksum calculation to an S3 object at rest, iRODS must read the object's data to calculate the checksum. The default buffer size for these reads is 1 MiB. Each of these read requests map to a separate GetObject request to S3 with a range of bytes. This can exceed the request rate that the S3 provider allows.
+
+To resolve this problem, the read buffer size can be increased by updating the `checksum_read_buffer_size_in_bytes` value in the `advanced_settings` stanza of server_config.json.
```

Comment:
Maybe mention that the setting needs to be updated on all servers which can service the S3 resource requests? Maybe that's implied, but could be good to say out loud (especially for the "detached" use cases).

Reply:
Good point.

Reply:
done